test/integration.html

```diff
@@ -43,14 +37,14 @@
       </iron-query-params>
     </template>
     <script type="module">
-import '@polymer/polymer/polymer-legacy.js';
-import '@polymer/promise-polyfill/promise-polyfill.js';
-import '../iron-query-params.js';
-import '../iron-location.js';
-import './replace-state-patch.js';
-import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
-Polymer({is: 'integration-element'});
-</script>
+      import '@polymer/polymer/polymer-legacy.js';
+      import '@polymer/promise-polyfill/promise-polyfill.js';
+      import '../iron-query-params.js';
+      import '../iron-location.js';
+      import './replace-state-patch.js';
```

Comment:
Could you move the imports that are duplicated here and in the script below up into a script in the head?